packages/shared/components/inputs/Amount.svelte

```diff
@@ -112,7 +112,7 @@
 
         // IOTA -> FIAT
         if (isFiatCurrency(toUnit)) {
-            amount = parseFloat(convertAmountToFiat(amount).slice(2))
+            amount = parseFloat(convertAmountToFiat(amount).slice(2)).toString()
```

Comment:
This should fix the NaN issue..
```suggestion
            let _amount = parseFloat(convertAmountToFiat(amount).slice(2)) ?? 0
            amount = isNaN(_amount)  ? '0' : _amount.toString() 
```